### PR TITLE
Create traditional-bolognese-sauce.md

### PR DESCRIPTION
**Pull request by yiusa**

### CODE REVIEW

**Comment by marttoygoy4** [comment]
From which cultrue? You cant just say "traditional"

**Comment by yiusa** [comment]
> From which cultrue? You cant just say "traditional"

Italian ofcourse, its where bolognese sauce comes from?

File: src/traditional-bolognese-sauce.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
+# Traditional bolognese sauce
+
+Basic bolognese sauce for lasagne or pasta dishes
+
+- ⏲️ Prep time: 45 min
+- 🍳Cook time: atleast 4 hours
+
+
```
Comment (hallzy) [inline src/traditional-bolognese-sauce.md:8]:
```suggestion
- 🍳 Cook time: at least 4 hours

```

File: src/traditional-bolognese-sauce.md [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+- 1 can peeled tomatoes 
+- 1-2 glasses wine (whatever you have open)
+- 1 glass of milk (optional)
+- 1 glass of beef/vegetable stock (optional)
+
+## Directions
+
+1. Finely dice the carrots, celery, onion and pancetta, if your blender can handle it you can optionally blend all this to speed it up.
+2. Heat up a large saucepan or dutch oven on medium low heat and put enough olive oil in to cover the bottom of the pan.
+3. Put in your diced vegies and pancetta and let it slowly sauté for about 10 minutes until everything is fragrant and soft.
```
Comment (hallzy) [inline src/traditional-bolognese-sauce.md:32]:
```suggestion
3. Put in your diced veggies and pancetta and let it slowly sauté for about 10 minutes until everything is fragrant and soft.
```

File: src/traditional-bolognese-sauce.md [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+4. If you are using garlic add it here and let it sweat for a minute
+5. Add in your minced meat and break it up into small clumps about the size of a rice grain with a spoon, keep stiring until all your meat is lightly cooked.
+6. Once all your meat is cooked and looks a bit grey add your wine, add enough to fully deglaze the pan (i usually add a little bit extra for the flavor)
```
Comment (hallzy) [inline src/traditional-bolognese-sauce.md:35]:
```suggestion
6. Once all your meat is cooked and looks a bit grey add your wine, add enough to fully deglaze the pan (I usually add a little bit extra for the flavor) 
```

File: src/traditional-bolognese-sauce.md [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+
+## Directions
+
+1. Finely dice the carrots, celery, onion and pancetta, if your blender can handle it you can optionally blend all this to speed it up.
+2. Heat up a large saucepan or dutch oven on medium low heat and put enough olive oil in to cover the bottom of the pan.
+3. Put in your diced vegies and pancetta and let it slowly sauté for about 10 minutes until everything is fragrant and soft.
+4. If you are using garlic add it here and let it sweat for a minute
+5. Add in your minced meat and break it up into small clumps about the size of a rice grain with a spoon, keep stiring until all your meat is lightly cooked.
+6. Once all your meat is cooked and looks a bit grey add your wine, add enough to fully deglaze the pan (i usually add a little bit extra for the flavor) 
+stir it through until it has completely evaporated.( if you are using it you can also add the stock at this point)
```
Comment (hallzy) [inline src/traditional-bolognese-sauce.md:36]:
```suggestion
stir it through until it has completely evaporated. (If you are using it you can also add the stock at this point)
```

File: src/traditional-bolognese-sauce.md [hunk 5]
```diff
@@ -0,0 +1,50 @@
...
+6. Once your pan is somewhat dry you can add the tomato paste to the beef, let it cook for a minute to remove the slight metalic taste.
```
Comment (hallzy) [inline src/traditional-bolognese-sauce.md:37]:
```suggestion
6. Once your pan is somewhat dry you can add the tomato paste to the beef, let it cook for a minute to remove the slight metallic taste.
```

File: src/traditional-bolognese-sauce.md [hunk 6]
```diff
@@ -0,0 +1,50 @@
...
+7. Add your canned and peeled tomatoes and use your spoon to break them up, you don't have to be too thorough since they will soften up while the sauce cooks.
+8. add the diced tomatoes, fill the jar your tomatos came in with water give it a shake and add that to the pot, do the same with your can of peeled tomatoes.
+9. lower your heat to the lowest setting and slightly cover the pan with your lid.
+10. let it simmer for as long as you can wait, preferably atleast 4 hours, if it starts getting a bit too dry you can add a bit more water to sure the sauce doesn't burn.
```
Comment (hallzy) [inline src/traditional-bolognese-sauce.md:41]:
```suggestion
10. let it simmer for as long as you can wait, preferably at least 4 hours, if it starts getting a bit too dry you can add a bit more water to make sure the sauce doesn't burn.
```

File: src/traditional-bolognese-sauce.md [hunk 7]
```diff
@@ -0,0 +1,50 @@
...
+11. once its done simmering for as long as you can muster and its nice and thick, the liquid should not come above your meat and should be pretty concentrated
```
Comment (hallzy) [inline src/traditional-bolognese-sauce.md:42]:
```suggestion
11. Once it's done simmering for as long as you can muster and its nice and thick, the liquid should not come above your meat and should be pretty concentrated
```

File: src/traditional-bolognese-sauce.md [hunk 8]
```diff
@@ -0,0 +1,50 @@
...
+you can add your glass of milk, let it simmer for 10 more minutes and serve.
```
Comment (hallzy) [inline src/traditional-bolognese-sauce.md:43]:
```suggestion
12. You can add your glass of milk, let it simmer for 10 more minutes and serve.
```

**Pull request merged**